src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationService.java

```diff
@@ -79,51 +80,46 @@ private EmailData prepareEmailData(Claim claim, Optional<String> defendantPin, S
 
     private List<EmailAttachment> getAttachments(
         final Claim claim,
-        final Optional<String> defendantPin,
-        final String submitterEmail
+        final String defendantPin,
+        final String submitterEmail,
+        final String authorisation
     ) {
         final List<EmailAttachment> emailAttachments = new ArrayList<>();
 
         if (!claim.getClaimData().isClaimantRepresented()) {
-            String pin = defendantPin.orElseThrow(NullPointerException::new);
-            emailAttachments.add(sealedClaimPdf(claim, submitterEmail));
+            String pin = Optional.ofNullable(defendantPin).orElseThrow(NullPointerException::new);
+            emailAttachments.add(sealedClaimPdf(authorisation, claim, submitterEmail));
             emailAttachments.add(defendantPinLetterPdf(claim, pin));
         } else {
-            emailAttachments.add(sealedLegalClaimPdf(claim));
+            emailAttachments.add(sealedLegalClaimPdf(authorisation, claim));
         }
 
         return emailAttachments;
     }
 
-    private EmailAttachment sealedLegalClaimPdf(Claim claim) {
-        byte[] generatedPdf = pdfServiceClient.generateFromHtml(
-            staffEmailProperties.getEmailTemplates().getLegalSealedClaim(),
-            legalSealedClaimContentProvider.createContent(claim)
-        );
+    private EmailAttachment sealedLegalClaimPdf(final String authorisation, final Claim claim) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "authorisation". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1172)